docs/pages/blog/nextgen-pickers.md

```diff
@@ -0,0 +1,130 @@
+It can recognize what's a month, day, or whatever piece of the date you are currently editing and act accordingly.
+
+For example, watch what happens when you edit the month with arrow keys.
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/nextgen-pickers/intelligent-field.mp4" type="video/mp4" />
+</video>
+
+As you increase the month, the component automatically watches for valid values throughout each piece of your date input, like you're navigating in a calendar.
+
+The Fields are a perfect match to the calendar view, but they are available as stand-alone components, and you can import them from the latest v6 package.
+
+```jsx
+import { Unstable_DateField as DateField } from '@mui/x-date-pickers/DateField';
+
+<DateField label="My first field" />;
+```
+
+They are also included by default on the new pickers.
+
+```jsx
+import { Unstable_NextDatePicker as NextDatePicker } from '@mui/x-date-pickers/NextDatePicker';
+
+<NextDatePicker label="My first v6 picker" />;
+```
+
+## Enhanced keyboard usability
+
+With the new fields, not only can you type the date as a text, but you can use your arrow keys to quickly navigate through each piece of date and time values.
+You can edit them atomically, without, for instance, leaking the year into the month, a common occurrence with masked inputs.
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/nextgen-pickers/keyboard-nav.mp4" type="video/mp4" />
+</video>
+
+For a quick comparison, let's check out, for instance, how the two different approaches behave on the most typical scenarios.
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/nextgen-pickers/fields-common-usecases.mp4" type="video/mp4" />
+</video>
+
+Try out the live demo of the `DateField` below.
+
+<iframe
+  src="https://codesandbox.io/embed/date-field-demo-pb87v0?fontsize=12&hidenavigation=1&module=%2F"
+  style="width:100%; height:200px; border:0; border-radius: 4px; overflow:hidden;"
+  allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+  sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+></iframe>
+
+### Fields roadmap
+
+The Fields are still in alpha, and there are many improvements on the way. The following list describes some of the most noticeable.
+
+- **Auto section switching**
+
+  So users don't need to manually navigate to the next section of the date after fulfilling the current one.
+
+- **Smart pasting**
+
+  Date strings pasted on Fields are parsed and matched with the sections of the date.
```

Comment:
Future tense since this is coming soon.

```suggestion
  Date strings pasted in Fields will be parsed and matched with the sections of the date.
```